container/tomcat/src/main/java/org/jboss/modcluster/container/catalina/standalone/ModClusterListener.java

```diff
@@ -19,12 +19,13 @@
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
-package org.jboss.modcluster.container.jbossweb;
-
+package org.jboss.modcluster.container.catalina.standalone;
 
 /**
- * @author Paul Ferraro
+ * Compatibility class.
  *
+ * @author Radoslav Husar
+ * @version Jun 2016
  */
-public class ServerTestCase extends org.jboss.modcluster.container.catalina.ServerTestCase {
+public class ModClusterListener extends org.jboss.modcluster.container.tomcat.ModClusterListener {
```

Comment:
OK for the @Deprecated ;-)
But changing "container-spi" to container/spi should go in another PR this one is already huge, no?